platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ExportMigrationEntryImpl.java

```diff
@@ -380,7 +380,13 @@ private boolean handleStoreFile() {
       // here it means the error came out of reading/processing the input file/stream
       // where it is safe to continue with the next entry, so don't abort
       SecurityLogger.audit(ExportMigrationEntryImpl.ERROR_EXPORTING_FILE, absolutePath);
-      getReport().record(newError(ExportMigrationEntryImpl.FAILED_TO_BE_EXPORTED, e));
+
+      // Will refrain from throwing an exception if there is a duplicate file that is being exported
```

Comment:
❗️ I am really not a fan of this. It relies on parts of the message to assume it is a duplicate. That could happen in other circumstances and create problems. We also generate a warning when it ain't a problem to have 4 properties referencing the same file.

I am providing a starting point for a fix offline where the export property reference entries are actually referencing actually export entries via the context like done at import time.